### ISSUE TITLE
Can you add the option to export all the scripts at the same time?

### ISSUE THREAD

**Issue by duradara** [MathewSachin/Fate-Grand-Automata#276]
Right now to export them we need to go one by one and its a little annoying. 
Its not really a super necessary feature but it will be nice to have it.

**Comment by MathewSachin** [comment]
Right now when exporting a single config, you are asked to specify the folder where it is to be saved and the file name.
How should that look for multiple files?

**Comment by duradara** [comment]
You just need to specify the folder. The scripts names can be the name they already have. 
Sorry if my english is bad

**Comment by MathewSachin** [comment]
If the scripts already exist in that folder, then they should be overridden, right?

**Comment by duradara** [comment]
Yes but maybe you could add a little warning telling people that they have a script with the same name in that folder.
